client/ayon_resolve/api/plugin.py

```diff
@@ -454,6 +456,13 @@ def convert(self):
         else:
             self.tag_data["asset"] = self.ti_name
 
+        # AYON unique identifier
+        folder_path = "/{}/{}".format(
+            self.tag_data["hierarchy"],
+            self.tag_data["asset"],
+        )
+        self.tag_data["folder_path"] = folder_path
```

Comment:
Does this have to be set on created instance? This can be collected during publishing.

Reply:
just to flag if we rename it "folderPath" right now, it creates invalid-context instances in the publisher.

Reply:
I think he cannot rename it to `folderPath` because publishe UI would complain that it does not exist.

Reply:
OK, so I think I've this:

* I temporarily stored what used to be set in `folder_path` into a temporary `target_folder_path` entry, so I do not overwrite `folderPath` which contains the value needed to keep the instance valid but store it in the creator_attributes instead. So `target_folder_path` will not be part of the instance.
* Then, as per @jakubjezek001 recommendation, I've implemented the replication via `editorialSharedData` into the shot collector so that `folderPath` is properly set for each product upon collection time (same logic than traypublisher).

![image](https://github.com/user-attachments/assets/a53c5966-0643-473f-a53f-b2a22337aab8)